README.md

```diff
@@ -281,8 +278,8 @@ SymCC is free software: you can redistribute it and/or modify it under the terms
 of the GNU General Public License as published by the Free Software Foundation,
 either version 3 of the License, or (at your option) any later version.
 
-As an exception from the above, you can redistribute and/or modify the SymCC
-runtime under the terms of the GNU Lesser General Public License as published by
+As an exception from the above, you can redistribute and/or modify the [SymCC
+Runtime](https://github.com/eurecom-s3/symcc-rt) under the terms of the GNU Lesser General Public License as published by
 the Free Software Foundation, either version 3 of the License, or (at your
 option) any later version. See #114 for the rationale.
```

Comment:
Since we're now pulling in the runtime from a separate repository and mentioning it in the table below, I think it should be fine to drop this paragraph: The runtime isn't an "exception from the above" anymore, it's simply an external dependency with a separate license :man_shrugging:

Reply:
Ok, perfect.